cdbot/cogs/help.py

```diff
@@ -1,11 +1,20 @@
-from discord import Colour, Embed
+from discord import Colour, Embed, utils
 from discord.ext import commands
 
 
 class EmbeddedHelpCommand(commands.HelpCommand):
     def __init__(self):
         super().__init__(command_attrs={'help': 'Gives detailed information about a command.'})
 
+    async def command_callback(self, ctx, *, command=None):
+        bot = ctx.bot
```

Comment:
Since bot is only used once, perhaps all instances of bot should be replaced with `ctx.bot`.